docs/run_test/test_spec.rst

```diff
@@ -0,0 +1,166 @@
+Test Specification
+==================
+
+This file lists all test cases' specifications.
+
+.. class:: LsVmBus
```

Comment:
Can it be generated automatically?

Reply:
Yes. This is generated by test_spec_gen.py.

Reply:
If it's auto generated, don't commit it to repo.

Reply:
I misunderstood your words. It is generated by test_spec_gen.py, but not auto-generated I guess. You need to run the .py script manually.

Reply:
It can be added somewhere, like conf.py. So the readthedocs can call it, generate it on the build time. it will be auto-gen.

Reply:
Will study how to do it.

Reply:
Added in conf.py